docs/resources/logbook/2024_lhc.md

```diff
@@ -35,6 +35,8 @@ The tables below can be sorted by clicking next to the column headers.
 | 2024-04-01 15:30 | 2024-04-01 23:00 |     1H     | Commissioning |                         Squeeze 30cm-22cm                          | [Shift Plan](2024-04-01, 1081, 3996950){.logbook-link} / [Summary](2024-04-01, 1081, 3997092){.logbook-link} |
 | 2024-04-09 12:30 | 2024-04-09 14:00 |    0.3W    | Commissioning |                 Measurements with Beam-Beam Wires                  | [Shift Plan](2024-04-09, 1081, 4002607){.logbook-link} / [Summary](2024-04-09, 1081, 4002706){.logbook-link} |
 | 2024-05-14 19:00 | 2024-04-15 04:00 |    1WN     |   MD 11743    | Effect of higher-order contributions to octupole and decapole RDTs | [Shift Plan](2024-05-14, 1081, 4067449){.logbook-link} / [Summary](2024-05-15, 1081, 4067625){.logbook-link} |
+| 2024-06-06 07:00 | 2024-06-06 14:00 | 0.5WN 0.4W |   MD 12466    |                  Effect of CMS Solenoid on Optics                  | [Shift Plan](2024-06-06, 1081, 4082214){.logbook-link} / [Summary](2024-06-06, 1081, 4082930){.logbook-link}  |
+| 2024-06-06 15:00 | 2024-06-07 01:25 | 0.5W 0.3WN |   MD 11243    |                        HL-LHC Optics Part 1                        | [Shift Plan](2024-06-06, 1081, 4082563){.logbook-link} / [Summary](2024-06-07, 1081, 4082920){.logbook-link} |
```

Comment:
I don't think I agree with your estimates.
MD 12466 is 
7:00 - 8:30 = 1.5/8 = 0.2 WN
8:30 - 14:00 = 5.5/8 = 0.7 W

MD 11243
15:00 - 17:30 = 2.5/8 =  0.3 W
17:30 - 01:30 = 8/8 = 1 WN

Reply:
Thanks, I can't math